.travis/benchmark-integration-test-verdaccio.sh

```diff
@@ -22,6 +22,7 @@ npm i && npm run repoclean -- --yes && npm run bootstrap
 
 # Call CLI through the local binary
 export CALL_METHOD="npx caliper"
+export GENERATOR_METHOD="yo"
```

Comment:
you will need the explicit generator call here ... ie `yo caliper-generator` etc